src/gluonts/model/rotbaum/_model.py

```diff
@@ -99,7 +100,7 @@ def __init__(
             true values associated with each prediction.
         """
         if model:
-            self.model = model
+            self.model = copy.deepcopy(model)
```

Comment:
Do we need a `deepcopy` here?

Reply:
This is copying an untrained model, so it's not a drain on resources. We do need it do deepcopy, because Rotbaum will call multiple QRX models (one for each timestep in the future), and we want each of them to be a different model trained on different data.